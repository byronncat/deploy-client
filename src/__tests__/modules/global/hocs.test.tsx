import { Global } from '@global';
import { render } from '@testing-library/react';

describe('hoc', () => {
  describe('global', () => {
    it('should render the global hoc correctly', () => {
      const { container } = render(<Global>Test</Global>);
      expect(container).toMatchSnapshot();
    });
  });
});
