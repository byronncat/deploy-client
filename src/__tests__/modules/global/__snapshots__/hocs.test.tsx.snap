// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hoc global should render the global hoc correctly 1`] = `
<div>
  Test
  <div
    class="Toastify"
  />
</div>
`;
